### ISSUE TITLE
Please add gfx906 to support Vega20 in cmakelists

### ISSUE THREAD

**Issue by rkothako** [ROCmSoftwarePlatform/rocPRIM#21]
Can you please add gfx906 to support Vega20 in cmakelists @ https://github.com/ROCmSoftwarePlatform/rocPRIM/blob/master/CMakeLists.txt#L70

**Comment by jszuppe** [comment]
Did you run it on Vega20?

**Comment by rkothako** [comment]
Yes, we run it on Vega20 also and hence we need Vega20 support. Please add.

**Comment by jszuppe** [comment]
Sure. Just wanted confirmation it works ok. I'll change it in `develop` today.

**Comment by jszuppe** [comment]
Check out https://github.com/ROCmSoftwarePlatform/rocPRIM/tree/develop-gfx906. It will be merged into `develop` when there's a public release of ROCm with support for gfx906.

**Comment by rkothako** [comment]
Superb...Let me use develop-gfx906 for Vega20 and comeback with observations.

**Comment by rkothako** [comment]
Thanks @jszuppe ,
I tested on Vega20 with develop-gfx906 branch, tests are running well.
But few failures observed. Looks like issue is due to compiler, but not sure.

91% tests passed, 8 tests failed out of 90

Total Test time (real) = 246.16 sec

The following tests FAILED:
          9 - rocprim.hc.block_radix_sort (Failed)
         24 - rocprim.hc.device_segmented_radix_sort (Failed)
         43 - rocprim.hip.block_radix_sort (Failed)
         51 - rocprim.hip.device_merge_sort (Failed)
         53 - rocprim.hip.device_radix_sort (OTHER_FAULT)
         58 - rocprim.hip.device_segmented_radix_sort (Failed)
         80 - hipcub.DeviceRadixSort (Failed)
         85 - hipcub.DeviceSegmentedRadixSort (Failed)
Errors while running CTest

**Comment by jszuppe** [comment]
We would need more info, but I doubt we can help without environment and gpu.

**Comment by rkothako** [comment]
Thanks @jszuppe 
Looks like issue is NOT specific to Vega20. Its observed in Vega10/MI25 also and we have logged an internal ticket, found to be compiler issue.

**Comment by jszuppe** [comment]
I'll rebase `develop-gfx906` on current `develop` later today.

**Comment by VincentSC** [comment]
Are all of these issues solved, including test 51?
